save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ProjectView.kt

```diff
@@ -559,7 +563,9 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
         div("row align-items-center justify-content-center") {
             nav("nav nav-tabs mb-4") {
                 ProjectMenuBar.values()
-                    .filter { it != ProjectMenuBar.SETTINGS || state.selfRole.isHigherOrEqualThan(Role.ADMIN) }
+                    .filter {
+                        (it != ProjectMenuBar.SETTINGS && it != ProjectMenuBar.RUN) || state.selfRole.isHigherOrEqualThan(Role.ADMIN)
+                    }
```

Comment:
Maybe, invert condition and use `filterNot`? I think it would be more clear that we forbid certain tabs for certain roles

Reply:
Oh ok

Reply:
To my mind there is no difference
```
.filterNot {
    (it == ProjectMenuBar.RUN || it == ProjectMenuBar.SETTINGS) && !state.selfRole.isHigherOrEqualThan(Role.ADMIN)
}
```
and
```
.filter {
    (it != ProjectMenuBar.SETTINGS && it != ProjectMenuBar.RUN) || state.selfRole.isHigherOrEqualThan(Role.ADMIN)
}
```

Reply:
For me the first one is 'We are filtering out elements if it's RUN or SETTING tab and role doesn't grant enough permissions'. And for the latter it's 'We are keeping elements if they are neither SETTINGS nor RUN OR if role is high enough'. Idk, it's just personal preference probably, but for me the first variant seems more clear and explicit.

Reply:
Ok, let it be so